### ISSUE TITLE
Error while loading entrypoint 'EntryPoint(name='isbn', value='papis.isbn:Importer', group='papis.importer')': No module named 'pkg_resources'.

### ISSUE THREAD

**Issue by homocomputeris** [papis/papis#913]
* papis version (`$ papis --version` or commit number):
* papis, version 0.13

# how to reproduce the issue
Command and output of `papis --log INFO <command>`:
```
papis --log INFO init
Error while loading entrypoint 'EntryPoint(name='isbn', value='papis.isbn:Importer', group='papis.importer')': No module named 'pkg_resources'.
Error while loading entrypoint 'EntryPoint(name='isbn', value='papis.isbn:explorer', group='papis.explorer')': No module named 'pkg_resources'.
Command 'init' is unknown!
Usage: papis [OPTIONS] COMMAND [ARGS]...

Error: No such command 'init'.

```

**Comment by alexfikl** [comment]
Hi! You seem to have two separate errors there:
1. The `pkg_resources` error is due to a missing `setuptools` dependency. You just need to install that in your environment.
2. The `init` error is because that command is not available in `papis 0.13`. You can check out the existing commands for that version in the [documentation](https://papis.readthedocs.io/en/stable/commands.html) or with a `papis --help`.

Hope that helps!

**Comment by homocomputeris** [comment]
Thanks!
